compose/preview.c

```diff
@@ -181,6 +181,8 @@ static void draw_preview(struct MuttWindow *win, struct PreviewWindowData *wdata
 
       content_lines++;
 
+      line = mutt_str_expand_tabs(line, &line_len, 4);
```

Comment:
Do we want to constant the 4 here? Or even use another value (please not 😄).

Reply:
Sorry, it really needs to be 8.
That's the default in terminals and that's how it's hard-coded in the Pager.